source/main/utils/setupLogging.js

```diff
@@ -25,7 +25,9 @@ export const setupLogging = () => {
   log.transports.console.level = isTest ? 'error' : 'info';
   log.transports.rendererConsole.level = isDev ? 'info' : 'error';
   log.transports.file.level = 'debug';
-  log.transports.file.maxSize = 20 * 1024 * 1024;
+  log.transports.file.maxSize = 5 * 1024 * 1024;
```

Comment:
On this way you will set all logs to 5MB even if is original Daedalu.log (not old) and that means that 1xDaedalus.log + 3xDeadelus.log-time() will have sum of 20MB.

Please ensure that Daedalus.log is 20MB and and only old logs are 5MB

Reply:
No this is wrong @tomothespian. ALL logs need to be max 5 MB in size.